From: Patrycja Rosa <alpine@ptrcnull.me>
Date: Sun, 13 Feb 2022 21:41:11 +0100
Subject: use libadwaita-1.0.0-alpha4

it's currently fixed to alpha2 in meson.build, but seems to work with alpha4

--- a/meson.build
+++ b/meson.build
@@ -23,1 +23,1 @@
-dependency('libadwaita-1', version: '== 1.0.0-alpha.2')
+dependency('libadwaita-1', version: '== 1.0.0-alpha.4')
